app/models/ncr/work_order.rb

```diff
@@ -8,6 +8,13 @@ def self.table_name_prefix
 
   EXPENSE_TYPES = %w(BA60 BA61 BA80)
   BUILDING_NUMBERS = YAML.load_file("#{Rails.root}/config/data/ncr/building_numbers.yml")
+  FY16 = Time.zone.parse('2015-10-01')
```

Comment:
share that one line? `FY16 = ...` ?

the logic for setting MAX_AMOUNT?

or share the whole `validates` block?

if the latter, I think the only way to do that would be with a Concern mix-in. Right now there are 2 different validation messages (the default, and our custom one in `Ncr::WorkOrder`). Do we want to have the same message across both classes?